static/app/components/modals/metricWidgetViewerModal.tsx

```diff
@@ -207,11 +198,41 @@ function MetricWidgetViewerModal({
   );
 }
 
-export function ContextMenu({
-  metricsQuery,
-}: {
-  metricsQuery: MetricsQuery;
-}) {
+function WidgetTitle({value, displayValue, placeholder, onSubmit}) {
+  const [isEditingTitle, setIsEditingTitle] = useState(false);
+  const [title, setTitle] = useState<string>(value);
+
+  return (
+    <WidgetTitleRow>
+      {isEditingTitle ? (
+        <Input
+          value={title}
+          placeholder={placeholder}
+          onChange={e => {
+            setTitle?.(e.target.value);
+          }}
+        />
```

Comment:
This input has styling issues in dark mode:
<img width="1191" alt="Screenshot 2024-02-19 at 10 04 34" src="https://github.com/getsentry/sentry/assets/7033940/adabce1d-c986-4a3a-b3cd-43515b904c4e">

Reply:
wrong component imported :/ Thanks!